adapter/src/datasource/model/DatasourceInsertStatement.ts

```diff
@@ -0,0 +1,14 @@
+export interface DatasourceInsertStatement {
```

Comment:
In general: please don't use type `any`!

If you need a type for something that is unknown, use type `unknown` (e.g. because we don't care). All other cases need a real type.